vhpidirect/quickstart/sharedvar/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+set -e
+
+cd $(dirname "$0")
+
+echo "Analyze pkg.vhd ent1.vhd ent2.vhd tb.vhd"
```

Comment:
According to the previous comment, we would generate three binaries:

- `pkg_93.vhd`
- `--std=08 -frelaxed pkg_93.vhd`
- `--std=08 pkg_08.vhd`

However, I don't know if the following sequence will work:

```sh
ghdl -a pkg_93.vhd ent.vhd tb.vhd
ghdl -a --std=08 pkg_08.vhd ent.vhd tb.vhd

ghdl -e -Wl,main.c tb_93
ghdl -e --std=08 -Wl,main.c tb_08

ghdl -a --std=08 -frelaxed pkg_08.vhd
ghdl -e --std=08 -frelaxed -Wl,main.c tb_08relaxed

./tb_93
./tb_08
./tb_08relaxed
```

We might need to write each `-a`, `-e`, execute sequence independently and clean artifacts between executions.

Context: ghdl/ghdl#980

Reply:
I clean the artifacts between. In particular I believe that one gets odd results if more than one work-obj exists:
```
pkg_93.vhd:1:1:warning: package "pkg" was also defined in file "pkg_08.vhd" [-Wlibrary]
ent_93.vhd:1:1:warning: entity "ent" was also defined in file "ent_08.vhd" [-Wlibrary]
```

Removing the clean up adds these warnings. So I'm leaving the clean up in.

Reply:
It's ok if cleaning is required. However, it should not be required to have `ent_93.vhd` and `ent_08.vhd`. It's exactly the same content, isn't it?

Reply:
They are different because in ent_08 I have to call the protected variable's set and get methods. In the -frelaxed compilation I use ent_93 with the  --std=08 option, and that is where it says that the pkg and ent have been defined before. They are just warnings but they're making the example a little messy. I could name each file's ent/pkg as the file is named. But that is beating around the bush.

Reply:
You are correct. What about adding, for completeness, a helper `procedure set(var: ...)` in both packages? That would provide a nice example about how to make some entity compatible with either package. Just add a comment (`--for compatibility`).

> They are just warnings but they're making the example a little messy.

The output looks good now (after cleaning everything betweens (sub)examples), isn't it?